apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -484,8 +485,8 @@ const TeacherApplication = React.createClass({
                           <td key={j} style={likertSurveyCell}>
                             <Radio
                               name={question}
-                              value={beliefPoll[j]}
-                              checked={this.state[question] === beliefPoll[j]}
+                              value={j + 1}
+                              checked={parseInt(this.state[question]) === j + 1}
```

Comment:
Always specify radix for [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt): `parseInt(this.state[question], 10)`